a10_neutron_lbaas/v2/handler_lb.py

```diff
@@ -86,6 +86,16 @@ def stats(self, context, lb):
                     "total_connections": 0
                 }
 
+    def oper(self, context, lb, lb_db, model_type):
```

Comment:
this should be _oper, but let's go one step further and make that name more self-documenting. I'd suggest "_update_operating_status".